packages/cli/src/util/config/files.ts

```diff
@@ -1,16 +1,16 @@
 import { join, basename } from 'path';
 import loadJSON from 'load-json-file';
-import writeJSON from 'write-json-file';
+import { writeJsonFileSync } from 'write-json-file';
```

Comment:
Since this is the only file in `cli` that uses `write-json-file`, perhaps we can remove `write-json-file` in favor of `fs-extra`'s `fs.outputJson()`.